contracts/oracles/AnkrBNBOracle.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+import { OracleInterface } from "../interfaces/OracleInterface.sol";
+import { IAnkrBNB } from "../interfaces/IAnkrBNB.sol";
+import { ensureNonzeroAddress } from "@venusprotocol/solidity-utilities/contracts/validators.sol";
+import { EXP_SCALE } from "@venusprotocol/solidity-utilities/contracts/constants.sol";
+
+/**
+ * @title AnkrBNBOracle
+ * @author Venus
+ * @notice This oracle fetches the price of ankrBNB asset
+ */
+contract AnkrBNBOracle is OracleInterface {
+    /// @notice Address of ankrBNB
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    address public immutable ankrBNB;
+
+    /// @notice Address of Resilient Oracle
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    OracleInterface public immutable RESILIENT_ORACLE;
+
+    /// @notice Set this as asset address for native token on each chain.
+    address public constant NATIVE_TOKEN_ADDR = 0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB;
+
+    /// @notice Constructor for the implementation contract.
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(address _ankrBNB, address resilientOracleAddress) {
+        ensureNonzeroAddress(_ankrBNB);
+        ensureNonzeroAddress(resilientOracleAddress);
+        ankrBNB = _ankrBNB;
+        RESILIENT_ORACLE = OracleInterface(resilientOracleAddress);
+    }
```

Comment:
If we are using underscore in the param name then we should use it in all the params.
```suggestion
    constructor(address _ankrBNB, address _resilientOracleAddress) {
        ensureNonzeroAddress(_ankrBNB);
        ensureNonzeroAddress(_resilientOracleAddress);
        ankrBNB = _ankrBNB;
        RESILIENT_ORACLE = OracleInterface(_resilientOracleAddress);
    }
```